Assets/Scripts/Internal/DaggerfallBillboard.cs

```diff
@@ -82,34 +79,31 @@ void Start()
                     // Example is the treasury in Daggerfall castle, some action records flow through the quest item marker
                     meshRenderer.enabled = false;
                 }
-            }
-        }
 
-        void OnDisable()
-        {
-            restartAnims = true;
-        }
+                // wait a single frame so all objects initialize
+                yield return null;
 
-        void Update()
```

Comment:
I got rid of `Update` and moved this code to `Start` but I worry a bit this could have introduced some kind of unforeseen consequence. If you can point me any cases which I should test and make sure it works, that would be great. It looks fine on my machine, but I want to make sure my changes won't break something that I didn't even consider.

Reply:
a8a36c44f5b61423dc50e18460d4dbbeff243db5 reverts these ill changes